app/src/renderer/components/transactions/LiBankTransaction.vue

```diff
@@ -6,21 +6,25 @@
   >
     <template v-if="address === ''">
       <div slot="caption">
-        Sent&nbsp;<b>{{ full(atoms(coins.amount)) }}</b>
-        <span>&nbsp;{{ coins.denom.toUpperCase() }}</span>
+        Sent<b>{{ full(atoms(coins.amount)) }}</b>
```

Comment:
I would keep a space between `Sent` and the amount

Reply:
that's a double space on the UI, @jbibla found

Reply:
Funny, it seems no differences at occurs by adding or removing it